cmp/grid/GridModel.js

```diff
@@ -728,6 +728,11 @@ export class GridModel extends HoistModel {
             .map(({colId, width, hidden, pinned}) => ({colId, width, hidden, pinned}));
     }
 
+    setColumnState(colState) {
+        colState = this.persistenceModel.cleanColumnState(colState);
```

Comment:
this will throw if there is no persistenceModel.  But its totally valid (and common) for a grid not to have a persistence model.  Also, needs jsdoc